include/OpenSpaceToolkit/Astrodynamics/Flight/System/Dynamics/AtmosphericDrag.hpp

```diff
@@ -99,6 +100,12 @@ class AtmosphericDrag : public Dynamics
 
     SatelliteSystem getSatelliteSystem() const;
 
+    /// @brief              Declares the coordinates involved in the contribution calculation
+    ///
+    /// @param              [in] coordinatesBroker The coordinate broker mapping state coordinates and indexes
+
+    virtual void declareCoordinates(const Shared<CoordinatesBroker>& coordinatesBroker) override;
```

Comment:
```suggestion
    void declareCoordinates(const Shared<CoordinatesBroker>& coordinatesBroker) override;
```

nit. Does this need to be virtual?

Reply:
It needs to be virtual since we're overriding it explicitly I believe.

Reply:
It is being overriden yes
(copied the virtual func pattern from `applyContribution`)